tests/test_dispatch_types.py

```diff
@@ -0,0 +1,160 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Tests for the frequenz.client.dispatch types."""
+
+from datetime import datetime, timedelta, timezone
+
+
+def test_component_selector() -> None:
+    """Test the component selector."""
+    # pylint: disable=import-outside-toplevel
+    from frequenz.client.common.microgrid.components.components import ComponentCategory
+
+    from frequenz.client.dispatch._types import (
+        ComponentSelector,
+        _component_selector_from_protobuf,
+        _component_selector_to_protobuf,
+    )
+
+    assert ComponentSelector is not None
```

Comment:
Why this assert? How could it be `None`? I don't get this.

I guess maybe is a copy&paste thing from the very silly tests in the generated files in the API? In there it makes *some* sense because we are "testing" generated code, and generated code that is super dynamic, so maybe it could happen that you import something and it is `None`, but not even sure if they make sense there either.